api/client/gro_client.py

```diff
@@ -129,6 +129,7 @@ def find_data_series(self, **kwargs):
             search_results.append(
                 self.search('regions', kwargs['partner_region'])[:cfg.MAX_RESULT_COMBINATION_DEPTH])
             keys.append('partner_region_id')
+        all_data_series = []
```

Comment:
@nemozen The changes in these files seem to come from another unrelated PR https://github.com/gro-intelligence/api-client/pull/127